concrete/src/Page/Collection/Version/Version.php

```diff
@@ -693,13 +694,21 @@ public function createNew($versionComments)
      * @param bool $doReindexImmediately reindex the collection contents now? Otherwise it's reindexing will just be scheduled
      * @param string|\DateTime|int|null $cvPublishDate the scheduled date/time when the collection is published (start)
      * @param string|\DateTime|int|null $cvPublishEndDate the scheduled date/time when the collection is published (end)
+     * @param bool $keepOtherScheduling Set to true to keep scheduling of other versions
+     *                                  (e.g., users can view the current live version until the publish date of this version).
+     *                                  Set to false (default) to disapprove all other versions
+     *                                  (e.g., users can't see this page until the publish date of this version, even if this page is live now).
+     *
+     * @since 9.0.0 Added $keepOtherScheduling argument
```

Comment:
@aembler 

What is the version number of the next release?  We should change this annotation